src/server/game/DataStores/DB2Metadata.h

```diff
@@ -4402,7 +4402,7 @@ struct SceneScriptMeta
 {
     static DB2Meta const* Instance()
     {
-        static char const* types = "sshh";
+        static char const* types = "ssii";
```

Comment:
uint16 (h) was correct here - this file is autogenerated, the only manual changes allowed are `s` to `S` (LocalizedString to char) conversions